lib/core/shell.rb

```diff
@@ -66,6 +66,16 @@ def self.should_hide_output?
     tmp_stderr && !verbose
   end
 
+  def self.cmd(cmd_to_run)
+    output = `#{cmd_to_run}`
```

Comment:
Switch to https://www.perplexity.ai/search/Does-this-give-NVuep054SB.LgaY0YZWCKA

```ruby
require 'open3'
stdout, stderr, status = Open3.capture3(cmd_to_run)
```

change places that used output to get both stdout and stderr